dadguide/database_manager.py

```diff
@@ -201,10 +219,79 @@ def _get_table_fields(self, table_name: str):
             raise DadguideTableNotFound(table_name)
         return fields, pk
 
-    def get_active_skill(self, active_skill_id: int):
+    def build_graph(self):
+        self.graph = nx.DiGraph()
+
+        ms = self._query_many("SELECT * FROM monsters", (), DictWithAttrAccess)
+        es = self._query_many("SELECT * FROM evolutions", (), DictWithAttrAccess)
+        aws = self._query_many("SELECT * FROM awakenings", (), DgAwakening)
+        lss = self._query_many("SELECT * FROM leader_skills", (), DgLeaderSkill, idx_key='leader_skill_id')
+        ass = self._query_many("SELECT * FROM active_skills", (), DgActiveSkill, idx_key='active_skill_id')
+        ss = self._query_many("SELECT * FROM series", (), DgSeries, idx_key='series_id')
+
+        for m in ms:
+            self.graph.add_node(m.monster_id, **m,
+                                    dgawakenings=[],
+                                    dgls=lss.get(m.leader_skill_id),
```

Comment:
would recommend not abbreviating, and also remove dg prefix cuz dadguide literally means jack shit